tests/ga/test_report_status.py

```diff
@@ -139,12 +139,11 @@ def http_get_handler(url, *_, **__):
             self._test_supported_features_includes_fast_track(protocol, False)
 
     def _test_supported_features_includes_fast_track(self, protocol, expected):
-        with patch("azurelinuxagent.common.conf.get_enable_fast_track", return_value=True):
```

Comment:
Removed all the patches that were setting EnableFastTrack to True. I made FastTrack the default several weeks ago, but was keeping these patches around in case we needed a release with FastTrack disable. No need to keep them around anymore.